Assignment/src/test/java/com/test/assignment/steps/firstAssignmentSteps.java

```diff
@@ -0,0 +1,62 @@
+package com.test.assignment.steps;
+
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.WebDriverRunner;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class firstAssignmentSteps {
+    @Given("user navigate to the qaTechHub page")
+    public void userNavigateToTheQaTechHubPage() {
+        System.out.println("starting the web browser");
+        open("https://qatechhub.com");
+        System.out.println("opening the qa tech hub page ");
+        //sleep(40000);
+        $(By.xpath("//h3[contains(text(),'QA Tech Hub: QA Automation Tools Trainings and Tut')]"));
```

Comment:
didn't understand what this line of code is doing, like is it for clicking or sending any value